src/classes/RD_AddDonationsBTN_CTRL.cls

```diff
@@ -30,13 +30,26 @@
 /**
 * @author Salesforce.com Foundation
 * @date 2012 (2.2)
+* @group Recurring Donations
```

Comment:
only one file in the group should include the group content tag, and I believe I have it in RD_RecurringDonations.cls